runtime/moonbase/tests/xcm_mock/statemint_like.rs

```diff
@@ -290,8 +288,37 @@ pub type Barrier = (
 parameter_types! {
 	pub MatcherLocation: Location = Location::here();
 	pub const MaxAssetsIntoHolding: u32 = 64;
+	pub const RelayTokenLocation: Location = Location::parent();
 }
 
+// Copied from:
+//
+// https://github.com/paritytech/polkadot-sdk/blob/f4eb41773611008040c9d4d8a8e6b7323eccfca1/cumulus
+// /parachains/common/src/xcm_config.rs#L118
+//
+// without the extra check for the system parachain, because we don't need it in tests.
+//
+// That checks ensures that our paraId is lower than ~1900.
+// If we keep it enabled our tests will fail given that our paraIds are 1, 2, 3 and so on.
```

Comment:
This comment does not apply to the code behaviour

Reply:
What do you mean exactly? In that comment I was referring to the original `ConcreteAssetFromSystem` struct. In that one, the second "System parachain" check ensures that the paraId is lower than 1900 aprox. 

In our case, if it matches a parachain (and as the mocked paraIds are 1, 2, 3) we need to return false, otherwise the destination parachain will be recognized as a system chain and AssetHub will try to teleport to them and thus will fail due to we don't allow teleports in our mocked parachains.

I can elaborate the comment a bit more if it's being confusing.

Reply:
Refactored the docs and the struct name in 3fe32b4edec990f87d32276cac356dec9a1816c0.